playfab-docs/sdks/unreal/quickstart.md

```diff
@@ -139,7 +145,6 @@ void ALoginActor::BeginPlay()
     Super::BeginPlay();
 
     clientAPI = IPlayFabModuleInterface::Get().GetClientAPI();
-    clientAPI->SetTitleId(TEXT("144"));
```

Comment:
I would not necessarily want to remove this line. I think there's a flavor of coders that prefer to set this in the Project Settings, and a flavor that prefer to set it from code. I personally think both are valid, and both should continue to be supported.

I would accept a change to this line to indicate it's one or the other, but would prefer not to remove it entirely.